modules/text2vec-cohere/vectorizer/objects.go

```diff
@@ -52,7 +52,7 @@ func New(client text2vecbase.BatchClient, logger logrus.FieldLogger) *text2vecba
 	}
 	// there does not seem to be a limit
 	maxTokensPerBatch := func(cfg moduletools.ClassConfig) int { return 500000 }
-	return text2vecbase.New(client, batch.NewBatchVectorizer(client, 50*time.Second, MaxObjectsPerBatch, maxTokensPerBatch, MaxTimePerBatch, logger), batchTokenizer)
+	return text2vecbase.New(client, batch.NewBatchVectorizer(client, 50*time.Second, MaxObjectsPerBatch, maxTokensPerBatch, MaxTimePerBatch, logger, "cohere"), batchTokenizer)
```

Comment:
let's add the name of the module to the client,

eg in module.go let's do
```
	m.vectorizer = vectorizer.New(client, m.logger, m.Name)
```

and then add a new method to the client interface. Otherwise there will be copy&paste errors :)

Reply:
I'll create a ticket for it :-)